spec/latest/json-ld-api/index.html

```diff
@@ -40,33 +40,48 @@
       githubAPI:              "https://api.github.com/repos/json-ld/json-ld.org",
       includePermalinks:      true,
       noRecTrack:             false,
-      testSuiteURIkey:       "http://json-ld.org/test-suite/", 
+      testSuiteURIkey:       "http://json-ld.org/test-suite/",
       postProcess:            [internalizeTermListReferences],
 
       // editors, add as many as you like
       // only "name" is required
       editors:  [
-          { name: "Markus Lanthaler", url: "http://www.markus-lanthaler.com/",
```

Comment:
To be honest, I'm not too happy about these changes given [how much work](https://github.com/json-ld/json-ld.org/graphs/contributors) I put into this